README.md

```diff
@@ -30,10 +30,11 @@ production version as necessary:
 
     gcloud config configurations create gotraining
```

Comment:
Is there any point leaving this configuration here? Without it, it just uses `default` which is probably fine if we are not using configurations at all.